src/com/booksaw/betterTeams/customEvents/BelowNameChangeListener.java

```diff
@@ -15,5 +15,5 @@
 	 * 
 	 * @param event The event that has been added to a team or removed from a team
 	 */
-	public void run(BelowNameChangeEvent event);
```

Comment:
This is a part of the legacy API so will be called from outside the plugin so will require the public access modifier
I have added @depreciated annotation and readded the public